### ISSUE TITLE
Google Play Version Check

### ISSUE THREAD

**Issue by mikemike396** [eggheadgames/Siren#45]
Any reason why this library hasn't implemented the ability to check against google play similar to this library (https://github.com/javiersantos/AppUpdater)? If this was added it would match the iOS like library (Harpy and Siren - Swift). Having a json file on my server and having to update that just for the Android version doesn't seem ideal when the iOS version just works out of the box. Just curious thanks.

**Comment by mikemee** [comment]
In short, I couldn't find a reliable & scalable way to do this.   As mentioned in the README:

> Unfortunately, the Google public API for version checking requires a token, and due to logistics and rate limiting, it's not feasible to use the API from an Android app.

I took a look at AppUpdater just now (which [we link to in the README](https://github.com/eggheadgames/Siren#similar-android-libraries)), it appears to do this by reading the web page for the product and then searching for the text string with the version information. I remember deciding that this was too fragile (what if the format changes and then all my old apps can't detect and prompt for updating?), and it also has some weird edge cases to do with incompatible devices & versions (I can't remember where I found the details, but I remember researching and discovering it's a can of worms).

> Having a json file on my server and having to update that just for the Android version doesn't seem ideal when the iOS version just works out of the box.

Yes, when I created this library, I really hoped it would be an option, but I couldn't find one that I felt I could depend on, so the `json` file became the least worst solution.
